src/app/events/event-components/event-groups-list/event-groups-list-action-data.ts

```diff
@@ -0,0 +1,20 @@
+import { ACTIONS_VIEW_OPTION } from '../../../_constants/actions';
+import { NavItem, NavItemRole } from '../../../_models/common';
+
+export const viewOptionsDefs: NavItem[] = [
+  {
+    role: NavItemRole.ACTION,
```

Comment:
I think if you define role as NavItemRole.LINK and adjust the action constant (for example: ACTION_VIEW_OPTION.EVENTS to have value of '/events/list' ) it should work directly without additional work

Reply:
Resolved